Undefined reference fmt::v5::format_error::~format_error() when building dll

Hi, same setup as in #1204.

I get multiple
> \<file>: (.text+0x2c): undefined reference to `fmt::v5::format_error::~format_error()'

The destructor was added in commit 1c3197b8d84884b0e16ba76b2c211ac35d2fa679 defined in `format-inl.h`, but in <file> I only `#include <fmt/format.h>`. I think it should be added there, or the class has to be exported.

--- a/include/fmt/core.h
+++ b/include/fmt/core.h
@@ -401,7 +401,7 @@ struct error_handler {
   FMT_CONSTEXPR error_handler(const error_handler&) {}
 
   // This function is intentionally not constexpr to give a compile-time error.
-  FMT_API FMT_NORETURN void on_error(const char* message);
+  FMT_NORETURN FMT_API void on_error(const char* message);
 };
 }  // namespace internal
 

--- a/include/fmt/format-inl.h
+++ b/include/fmt/format-inl.h
@@ -219,8 +219,8 @@ FMT_FUNC Char internal::thousands_sep_impl(locale_ref) {
 }
 #endif
 
-FMT_FUNC format_error::~format_error() FMT_NOEXCEPT {}
-FMT_FUNC system_error::~system_error() FMT_NOEXCEPT {}
+FMT_API FMT_FUNC format_error::~format_error() FMT_NOEXCEPT {}
+FMT_API FMT_FUNC system_error::~system_error() FMT_NOEXCEPT {}
 
 FMT_FUNC void system_error::init(int err_code, string_view format_str,
                                  format_args args) {

--- a/include/fmt/format.h
+++ b/include/fmt/format.h
@@ -417,7 +417,7 @@ typedef basic_memory_buffer<char> memory_buffer;
 typedef basic_memory_buffer<wchar_t> wmemory_buffer;
 
 /** A formatting error such as invalid format string. */
-class format_error : public std::runtime_error {
+class FMT_API format_error : public std::runtime_error {
  public:
   explicit format_error(const char* message) : std::runtime_error(message) {}
   explicit format_error(const std::string& message)
@@ -2198,7 +2198,7 @@ class arg_formatter : public internal::arg_formatter_base<Range> {
  An error returned by an operating system or a language runtime,
  for example a file opening error.
 */
-class system_error : public std::runtime_error {
+class FMT_API system_error : public std::runtime_error {
  private:
   FMT_API void init(int err_code, string_view format_str, format_args args);
 

